src/text-input/tests/functional/TextInput.ts

```diff
@@ -120,22 +123,36 @@ registerSuite('TextInput', {
 			.click()
 			.type(validText)
 			.end()
-			.sleep(10)
-			.getProperty('className')
-			.then((className: string) => {
-				assert.notInclude(className, css.invalid);
-				assert.include(className, css.valid);
-			})
-			.findByCssSelector(`.${css.input}`)
-			.type(invalidText)
 			.end()
-			.sleep(10)
-			.getProperty('className')
-			.then((className: string) => {
-				assert.notInclude(className, css.valid);
-				assert.include(className, css.invalid);
+			.then(
+				pollUntilTruthy(
+					function(validClass) {
+						return !!document.querySelector(`#example-validated .${validClass}`);
+					},
+					[css.valid],
+					CLICK_WAIT_TIMEOUT,
+					POLL_INTERVAL
+				)
+			)
+			.then((isEqual) => {
```

Comment:
Looking at line 129 above, if you had to name that function, what would be a good name? I think `isEqual` on this line could use a better name, perhaps something like `is<Line29FunctionName>`

Reply:
Maybe `doesValidClassExist`?